com.thincast.client.json

```diff
@@ -0,0 +1,337 @@
+{
+    "app-id": "com.thincast.client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "rdc",
+    "separate-locales": false,
+    "tags": [
+        "proprietary"
+    ],
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pcsc",
+        /* Needs to talk to the network */
+        "--share=network",
+        /* Play sounds redirected from guests */
+        "--socket=pulseaudio",
+        /* Allow rw access to download folder */
+        "--device=all",
+        "--filesystem=home",
```

Comment:
The comment above says _download folder_. We generally prefer filesystem permissions to be narrowed down as they're easy to be modified by users, should they want it.

Reply:
updated the comment. The access is only used when the user selects a folder to redirect in the app and only allowing access to `Downloads` might be a bit too strict